src/core/plugins/spec/selectors.js

```diff
@@ -374,6 +374,9 @@ export function parameterValues(state, pathMethod, isXml) {
   let paramValues = operationWithMeta(state, ...pathMethod).get("parameters", List())
   return paramValues.reduce( (hash, p) => {
     let value = isXml && p.get("in") === "body" ? p.get("value_xml") : p.get("value")
+    if (Array.isArray(value) || List.isList(value)) {
```

Comment:
Could you explain why we need both of these checks? Would the check for `array` not be enough?

```suggestion
    if (Array.isArray(value)) {
```

Reply:
It was what I do in a first step, because when I play the "Try it out" in the GUI, value received is well an array.  
But when a write the unit test, value received is a List (array is converted to a list when calling fromJS).  
I was not sure we never encouter it using the GUI, that why I prefer to check array and list, and that's why i add the test with spec2 to generate a parameter value with a type array

Reply:
Thanks, you are right, tested it on my end.